components/calendar/calendar.js

```diff
@@ -533,7 +537,11 @@ class Calendar extends LocalizeCoreElement(RtlMixin(LitElement)) {
 			if (prop === '_shownMonth' && this._keyboardTriggeredMonthChange) {
 				this._focusDateAddFocus();
 			} else if (prop === 'selectedValue' && this.selectedValue) {
-				await this._updateFocusDateOnChange();
+				if (!this._focusDate || !checkIfDatesEqual(this._focusDate, getDateFromISODate(this.selectedValue))) {
```

Comment:
Minor change but it may be nice to split this line into two to make the comparison easier to read. Something like
```suggestion
				const selectedDate = getDateFromISODate(this.selectedValue);
				if (!this._focusDate || !checkIfDatesEqual(this._focusDate, selectedDate)) {
```